### ISSUE TITLE
Tray icon new email notification not showing

### ISSUE THREAD

**Issue by IgorKvasn** [nylas/N1#701]
Hi,
I noticed that *sometimes* tray icon does not indicate new email even though I heard sound notification for incoming email (I even wait for couple of minutes).
However when I open N1 main window, the tray icon immidiately changes to indicate new email.

as a second issue (probably related to what I mentioned above):
I also noticed that when I have multiple email accounts linked to N1 and I send an email from one account to another, the tray icon or any kind of notification is shown - I am not sure if this is intentional or not....

observed on Windows and Linux
version 0.3.29

**Comment by IgorKvasn** [comment]
Actually I think I found the main problem here - tray notification changes based on the currently selected email account.
I've got 3 accounts linked to N1 and one of them has a new (unread) email -> when I switch to this account, a tray notification shows, but when I switch to any other account, the notification is gone (btw. by tray notification I mean the blue dot in the tray icon... I don't know how else to name it :)

I guess the notification should be visible independently of the selected account, right?
